Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -643,12 +644,13 @@ void UpdateMenu()
 
 			foreach (ToolbarItem item in _toolbarTracker.ToolbarItems)
 			{
+				IMenuItemController controller = item;
```

Comment:
*Suggestion* Could expand on this pattern to replace `((IController)Element).ControllerMethod` to a property `IController Controller => this` and then the call sites could simply be `Controller.ControllerMethod`.